docs/deployment.md

```diff
@@ -0,0 +1,52 @@
+# Deployment
+
+## Requirements
+
+- K8s cluster
+- Helm
+- Ingress controller (e.g. Traefik)
+- Nvidia GPU Operator
```

Comment:
The last three are recommendations. You can install without Helm, you don't need ingress if you are serving the model internally and you can use CPU for embeddings.

Reply:
Done